src/canopy_bioemi_mod.F90

```diff
@@ -91,7 +101,19 @@ SUBROUTINE CANOPY_BIO( ZK, FCLAI, FCH, LAI, FSUN, PPFD_SUN, &
         REAL(RK) :: CT1                            ! Activation energy (kJ/mol)
         REAL(RK) :: CEO                            ! Empirical coefficient
         REAL(RK) :: EF                             ! Final Mapped Emission factor (EF) (ug/m2 hr)
+
+        REAL(RK) :: TABOVECANOPY(SIZE(ZK))  ! Above Canopy Temp (assigned = TLEAF_AVE for now)
```

Comment:
Why is TABOVECANOPY indexed on the canopy levels, and shouldn't it just be set to the above canopy air temperature, as is provided by GFS met model here for example?  TMP2M is an input variable to canopy-app, and you could add it as an input to bioemi, as is similarly passed to the [tleaf module](https://github.com/quaz115/canopy-app-forkqzr/blob/develop/src/canopy_tleaf_mod.F90#L34).